src/HealthChecks.Kafka/KafkaHealthCheck.cs

```diff
@@ -8,32 +8,34 @@ namespace HealthChecks.Kafka
 {
     public class KafkaHealthCheck : IHealthCheck
     {
-        private readonly ProducerConfig _configuration;
+        private readonly IProducer<string, string> _producer;
+
         public KafkaHealthCheck(ProducerConfig configuration)
         {
-            _configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));

```

Comment:
Hi @rguyton 

We prefer throw expressions instead the if(null) throw, in this case using discards

Reply:
`_ = configuration ?? throw new ArgumentNullException(nameof(configuration));`